static/app/views/replays/detail/memoryChart.tsx

```diff
@@ -50,9 +61,9 @@ function MemoryChart({memorySpans = []}: Props) {
       min: 0,
       // we don't set a max because we let echarts figure it out for us
       axisLabel: {
-        formatter: value => formatBytesBase2(value),
+        formatter: value => getRoundedMemorySizeFromBytes(value),
```

Comment:
I think we probably want to use the `interval` setting: https://echarts.apache.org/en/option.html#xAxis.axisLabel.interval

What we've got here looks like we're re-labelling the tick marks, which doesn't re-position them to match the new number.

Reply:
I think I need some clarification here. If I use only the interval I don't get the formatting of the values to be rounded, I can go back to decimal formatting but that doesn't change the interval labels.

Reply:
I think we'll need both parts; `formatter` and `interval`.

Formatter is for sure needed to get the unit suffix for each label.
But interval is important too because it sets the distance between each label.

Before it was possible to have weird distances: 
<img width="88" alt="Screen Shot 2022-04-29 at 1 41 03 PM" src="https://user-images.githubusercontent.com/187460/166066343-56b847f4-4123-4174-b62a-ffc9090d5cff.png">
`19.1` to `28.6` is a distance of of `9.5`

For example sake lets say the total heap size is smaller, so the distance is (instead of 9.5) `4.5`. We'd have labels like:
- `22.5 MiB`
- `18 MiB`
- `13.5 MiB`
- `9 MiB`

If we just round those labels it'll be weird:
- `20 MiB`
- `20 MiB`
- `10 MiB`
- `10 MiB`

We also want to make sure that each point in the chart aligns with the axes labels. Currently they do.
This example shows 5.2 between 4.8 and 5.7, slightly closer to the one on the bottom.
<img width="351" alt="Screen Shot 2022-04-29 at 1 49 38 PM" src="https://user-images.githubusercontent.com/187460/166067384-b5ef1ece-e7cf-4d20-96c9-e884f4931c07.png">

Reply:
I'll have to come back to this Monday, I think I almost have it but setting an interval doesn't change how the graph is rendering.

Reply:
Okay I think I got it.. I set the minInterval to 1 and a busier looking chart seems to be correct. 
<img width="1251" alt="image" src="https://user-images.githubusercontent.com/7014871/166244553-399da989-eea1-4fae-aff8-7d0a0ea19901.png">
Here's the same chart unrounded for reference. 
<img width="1863" alt="image" src="https://user-images.githubusercontent.com/7014871/166244608-98a5dfc8-121e-426f-9027-fdbef19c882d.png">

Reply:
So i'm still worried about this happening:

<img width="1412" alt="Screen Shot 2022-04-29 at 3 56 42 PM" src="https://user-images.githubusercontent.com/187460/166253638-e50ba019-8c59-4c21-97b0-059dbe4459af.png">

From: https://dev.getsentry.net:7999/organizations/sentry-emerging-tech/replays/sourcemapsio-replays:49798a94d812457ab4c2ce1f1d4d3532/#memory

But i've got the docs open to help sort out the specifics

Reply:
Chatted offline, learned about all the echarts options and @Danondso has some improvements coming.